test/test_transforms.py

```diff
@@ -532,7 +532,15 @@ def test_convert_image_dtype_float_to_float(self):
             for output_dtype in output_dtypes:
                 with self.subTest(input_dtype=input_dtype, output_dtype=output_dtype):
                     transform = transforms.ConvertImageDtype(output_dtype)
+                    transform_script = lambda image: torch.jit.script(  # noqa: E731
+                        F.convert_image_dtype(image, output_dtype)
+                    )
```

Comment:
torchscript returns a function with the same signature as the original function, so you can do something like
```suggestion
                    script_fn = torch.jit.script(F.convert_image_dtype)
```